README.md

```diff
@@ -141,6 +141,14 @@ You can install `brokeneck-react` in your React application and use the exported
 
 For example, you can provide it as an additional route inside your application.
 
+brokeneck-react has a number of peer dependencies:
+
+- @material-ui/core: ^4.11.2,
```

Comment:
I've aded the versions here, but let me know if they need to be removed, as I'm not sure there's a need to have them, and they could become out of sync.

Reply:
I think we're fine without versions. The thing is, npm/yarn will show a warning if you install packages requiring peer deps you don't have, so this shouldn't be necessary. In practice, nobody looks at those warnings so it's good to document them, but no need to be so specific to include versions.